docs/usage.rst

```diff
@@ -35,8 +35,8 @@ virtualfish) use the same Python version that ``virtualenv`` was installed
 under, which will usually be whatever your default system Python is.
 
 If you want to use something different in a particular virtualenv, just pass in
-the ``-p PYTHON_EXE`` argument to ``vf new``, where ``PYTHON_EXE`` is any Python
-executable, for example::
+the ``--python PYTHON_EXE`` (``-p`` for brevity) argument to ``vf new``, where
```

Comment:
In the FAQ session the `--python` argument is mentioned as a thing to look in the Usage session. Now here it is.